test/tests/LayerTest.coffee

```diff
@@ -180,6 +180,28 @@ describe "Layer", ->
 			#layer.computedStyle()["background-size"].should.equal "cover"
 			#layer.computedStyle()["background-repeat"].should.equal "no-repeat"
 
+		it "should cancel loading when setting image to null", (done) ->
```

Comment:
That's already here: https://github.com/koenbok/Framer/pull/372/files#diff-cfd025d974e2cdd9ef6b86a3bea2f84aR158